src/engine/scripting/emscripten/src/websg.h

```diff
@@ -88,6 +88,7 @@ typedef struct NodeProps {
   camera_id_t camera;
   skin_id_t skin;
   mesh_id_t mesh;
+  collider_id_t collider;
```

Comment:
Because this isn't in the base GLTF spec, it should go in an extension. This is how we add features over time without breaking scripts. In this case if we added this property to this type we'd end up having incorrect byte alignment and reading the first rotation element instead of a collider id in older scripts